docs/src/main/sphinx/release/release-431.md

```diff
@@ -0,0 +1,60 @@
+# Release 431 (27 Oct 2023)
+
+## General
+
+* Add support for [](/routines). ({issue}`19308`)
+* Add support for [](/sql/create-function) and [](/sql/drop-function) statements. ({issue}`19308`)
+* Add support for the `REPLACE` modifier to the `CREATE TABLE` statement. ({issue}`13180`)
+* Disallow a `null` offset for the {func}`lead` and {func}`lag` functions. ({issue}`19003`)
+* Improve performance of queries with short running splits. ({issue}`19487`)
+
+## Security
+
+* Support defining rules for procedures in file-based access control. ({issue}`19416`)
+* Mask additional sensitive values in log files. ({issue}`19519`)
+
+## JDBC driver
+
+* Allow configuration of executing statements with the legacy
```

Comment:
This doesn't say what's the impact. I would rephrase as something along the lines of:

> Improve latency for prepared statements for Trino versions that support `EXECUTE IMMEDIATE` when the `explicitPrepare` parameter to is set to `false`.

Reply:
Updated